Dockerfile

```diff
@@ -27,10 +27,10 @@ RUN \
     ./authenticate.js "${FOUNDRY_USERNAME}" "${FOUNDRY_PASSWORD}" cookiejar.json && \
     s3_url=$(./get_release_url.js cookiejar.json "${FOUNDRY_VERSION}") && \
     wget -O ${ARCHIVE} "${s3_url}" && \
-    unzip -d dist ${ARCHIVE} 'resources/*'; \
+    unzip -d -o dist ${ARCHIVE} 'resources/*'; \
```

Comment:
This modification doesn't address the issue in the description.  These changes modify the way "pre-baked" Docker images are built, not how the runtime system installs FoundryVTT when it is not found.